### PR TITLE
Update guest-accounts.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Formatting, typos, rewriting for clarity.

### CODE REVIEW

File: source/deployment/guest-accounts.rst [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-===================================
+================================
 
 Available in `Enterprise Edition E10 and higher <https://about.mattermost.com/pricing/>`__.
 
-Guest Accounts are a new role within Mattermost Enterprise editions which allow organizations to collaborate with individuals (such as vendors and contractors) outside of their organization.
-The Guest Account feature controls access to channels and other members within the organization.
-
-For example, an organization may use Guest Accounts to work on a website project with resources from an external design firm. Guest Accounts could also be used to collaborate with customers on a support issue.
+The Guest Account feature provides organizations with a way to collaborate with individuals (such as vendors and contractors) outside of their organization by controlling their access to channels and team members. For example, Guest Accounts can be used to collaborate with customers on a support issue or work on a website project with resources from an external design firm.
```
Comment (amyblais) [inline source/deployment/guest-accounts.rst:8]:
```suggestion
The Guest Account feature provides organizations a way to collaborate with individuals (such as vendors and contractors) outside of their organization by controlling their access to channels and team members. For example, Guest Accounts can be used to collaborate with customers on a support issue or work on a website project with resources from an external design firm.
```

File: source/deployment/guest-accounts.rst [hunk 2]
```diff
@@ -42,24 +38,22 @@ Guest Authentication
...
-and `SAML Single-Sign-On <https://docs.mattermost.com/deployment/sso-saml.html>`_.
 
-Converting a member user to a guest will not change the channels they are in; however they will be restricted from discovering additional channels and unable to direct message / group message users outside of the channels they are in. They can be added to channels by System Admins and other roles that have the correct permissions to invite guests.
+Before you proceed, ensure that the authentication method you wish to use is correctly configured on your server and enabled in Mattermost. For configuration steps and technical documentation, see `Active Directory/LDAP Setup <https://docs.mattermost.com/deployment/sso-ldap.html>`_ and `SAML Single-Sign-On <https://docs.mattermost.com/deployment/sso-saml.html>`_.
 
+Converting a member user to a guest will not change the channels they are in; however they will be restricted from discovering additional channels and unable to Direct Message/Group Message users outside of the channels they are in. They can be added to channels by System Admins and other roles that have the correct permissions to invite guests.
```
Comment (amyblais) [inline source/deployment/guest-accounts.rst:44]:
```suggestion
Converting a member user to a guest will not change the channels they are in. However, they will be restricted from discovering additional channels and are unable to Direct Message/Group Message users outside of the channels they are in. They can be added to channels by System Admins and other roles that have the correct permissions to invite guests.
```

File: source/deployment/guest-accounts.rst (hunk 1 above)
Comment (justinegeffen) [reply source/deployment/guest-accounts.rst:8]: How about "Guest Accounts are a way to collaborate..."?

Comment (justinegeffen) [inline source/deployment/guest-accounts.rst:8]:
```suggestion
Guest Accounts are a way to collaborate with individuals (such as vendors and contractors) outside of your organization by controlling their access to channels and team members. For example, Guest Accounts can be used to collaborate with customers on a support issue or work on a website project with resources from an external design firm.
```

**Review by amyblais — APPROVED** [review]

File: source/deployment/guest-accounts.rst (hunk 1 above)
Comment (wiersgallak) [reply source/deployment/guest-accounts.rst]: @justinegeffen I like the suggestion

**Review by wiersgallak — APPROVED** [review]
LGTM

**Pull request merged**